_translator-files/po/zh_CN/1-index.po

```diff
@@ -35,24 +35,39 @@ msgid ""
 "mTOtherPlatforms: 'other platforms'\n"
 "---\n"
 msgstr ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
```

Comment:
Jekyll keeps complaining about this line but I can't see what's wrong with it. The error message is:
`did not find expected key while parsing a block mapping at line 2 column 1`
@ann0see any ideas?

Reply:
Ok, got it. Missing quotation marks further below.